Sources/SwiftKuery/Table.swift

```diff
@@ -101,32 +139,107 @@ open class Table: Buildable {
         return result
     }
     
-    /// Add alias to the table, i.e., implement the SQL AS operator.
-    ///
+    /// Function to create a String representation of a `Table` instance for use in an SQL CREATE TABLE query.
```

Comment:
Please can we get rid of the "Function that" or "Function to" wording from this file as it doesn't add anything which helps the user. Looking at Enrique's example it would then be "Returns the SQL CREATE TABLE query as a String"